configs/_base_/datasets/kitti_seg.py

```diff
@@ -0,0 +1,54 @@
+data_root = 'data/kitti-seg/'
+dataset_type = 'KittiSegDataset'
```

Comment:
```suggestion
dataset_type = 'KittiDataset'
```

Reply:
May rename to `kitti.py`

Reply:
Hi @xiexinch, the name of the dataset includes `seg` due to Kiiti dataset contains lots of different categories, such as object detection, and depth estimation, using *kittiseg* can clarify this dataset is especially for segmentation. 

See more on official website : http://www.cvlibs.net/datasets/kitti/

Reply:
As we will not add 'depth/flow estimation' or 'detection' task in mmseg, I think there may be no misunderstanding about 'KITTI'. Besides, in mmflow and mmdet3d, both of they use 'KITTI' install of 'KITTI flow' or 'KITTI det3d', so 'KITTI' is good to understand.

ref: 
https://github.com/open-mmlab/mmflow/blob/master/mmflow/datasets/kitti2015.py
https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/datasets/kitti_dataset.py